wikimedia/wikipedia-ios
# T376355 info icon

**Phabricator:** 
https://phabricator.wikimedia.org/T376355

### Notes
* None of the nice to have's in this PR, this will be part 1 if we want the nice to have's
* Also, updated UI bug re @tonisevener and updated UI according to Robin

### Test Steps
1. In English, Spanish, or Italian in Italy or Mexico in scheme
2. Open app, log in if not logged in
3. Enable year in review feature flag in developer settings
4. Navigate through 6+ articles, background and foreground app.
5. Navigate to Year in Review
6. Go into the data slides
7. Ensure information icon is there, and tapping it navigates user to the correct link for your language.
8. Repeat steps 5, 6, and 7 in another core language to ensure the link has the proper language. 


### Screenshots/Videos

https://github.com/user-attachments/assets/f45f2628-d569-4022-a0a4-c8917bbcadda
